### ISSUE TITLE
On Again Off Again

### ISSUE THREAD

**Issue by gringolalia** [yeoman/yeoman#1632]
I have a dual-boot working environment -- Linux Mint and Arch Linux (Apricity OS). 

The `yo` command works in the former context but not the latter, where I  actually prefer to work! 

The shell I use in Arch is zsh. Something to do with using `yo` in gnome-terminal?

**Comment by Munter** [comment]
Can you get more specific than "doesn't work" ?

**Comment by SBoudrias** [comment]
Yeah, do you see an error or anything?

**Comment by sindresorhus** [comment]
Support questions are better asked on Stack Overflow:
http://stackoverflow.com/questions/tagged/yeoman
Paste in the link to the question here so others can follow ;)

**Comment by bretonio** [comment]
Sorry, mucho vague. 

I have installed yo. I was obliged to install it with sudo npm i -g yo by the way. 

I then chose to install some generators. I then executed `yo` and the normal dialogue shows up only without the installed generators in the list. 

Then I try for example `yo hexo-theme` and am informed this generator "does not seem" to be installed. 

In ỳo doctor`I get a clean bill of health.  

`yo --help` shows that flask is installed and `yo flask` launches the generator. 

Tried un- and then re-installing from AUC (nodejs-yeoman) sans success.

For theme being will install with npm or clone projects to the other work environment on my machine where yo works like a charm.
